scripts/deploy.sh

```diff
@@ -56,12 +56,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # move OSA variables file to AIO location.
```

Comment:
What do you mean by official documentation? If you're referring to the documentation managed by the docs team, I am not aware of them ever documenting anything related to AIOs.

The point about having a better comment here is because the name of this file is important and if it's changed it has the potential the change the precedence. It breaks the new format of having defaults and overrides files and it isn't necessarily obvious why it's required. The idea of the comment is to prevent someone changing this section because they don't fully appreciate the context.